File: id/id/src/main/java/io/piano/android/id/PianoIdException.java
package io.piano.android.id;

/**
 * This class encapsulates exceptions, which were occurred at authorization process
 */
public class PianoIdException extends RuntimeException {
    public PianoIdException() {
        super();
    }

    public PianoIdException(String message) {
        super(message);
    }

    public PianoIdException(Throwable throwable) {
        super(throwable);
    }
}

File: id/id/src/main/java/io/piano/android/id/PianoIdException.kt
package io.piano.android.id

/**
 * This class encapsulates exceptions, which were occurred at authorization process
 */
public class PianoIdException : RuntimeException {
    public constructor() : super()
    public constructor(message: String?) : super(message)
    public constructor(throwable: Throwable?) : super(throwable)
}
